### DISCUSSION TITLE
Make Bubbles colours user configurable

### DISCUSSION THREAD

**Discussion by daniellekirkwood** [element-hq/element-meta#176] [Ideas]
As mentioned on https://github.com/vector-im/element-ios/issues/5420 a user has requested for Bubbles colours to be configurable by the user.

"Any chance this could be made user-configurable? I feel like it's super hard to agree on between people if chats should be blue/green/purple/black/gray/...

Maybe the user would just pick the main color and then that would be chosen for incoming, and a reduced saturation one for outgoing?"

**Comment by daniellekirkwood** [comment]
I particularly like the idea of picking one colour and the other colours in the bubbles palette building around that choice.

**Comment by SimonBrandner** [comment]
Should be definitely possible on web/desktop, though I wonder how would this interact with custom themes...
